src/core/issues-notes.js

```diff
@@ -141,22 +141,25 @@ export function run(conf, doc, cb) {
     issueBase = conf.issueBase;
   if ($ins.length) {
     if (conf.githubAPI) {
-      ghFetch(conf.githubAPI)
-        .then(function(json) {
-          issueBase = issueBase || json.html_url + "/issues/";
-          return fetchIndex(json.issues_url, {
-            // Get back HTML content instead of markdown
-            // See: https://developer.github.com/v3/media/
-            Accept: "application/vnd.github.v3.html+json",    
-          });
-        })
-        .then(function(issues) {
-          issues.forEach(function(issue) {
-            ghIssues[issue.number] = issue;
-          });
-          handleIssues($ins, ghIssues, issueBase);
-          cb();
-        }).catch(err => {
+      const asyncFetch = async () => {
+        const json = await ghFetch(conf.githubAPI);
+        const issues = await getIssues(json);
+        issues.forEach( (issue) => {
+          ghIssues[issue.number] = issue;
+        });
+        handleIssues($ins, ghIssues, issueBase);
+        cb();
+      }
+      const getIssues = (json) => {
+        issueBase = issueBase || json.html_url + "/issues/";
+        return fetchIndex(json.issues_url, {
+          // Get back HTML content instead of markdown
+          // See: https://developer.github.com/v3/media/
+          Accept: "application/vnd.github.v3.html+json",    
+        });
+      }
+      asyncFetch()
+        .catch(err => {
```

Comment:
Yes, I tried testing this, it works as of now, i.e., the promise returned was chained with the catch. Sorry for pointing that out

Reply:
NP :)